int32 num_rovers
int32 num_boxes
geometry_msgs/Point[] rover_pos
geometry_msgs/Point[] box_pos
